util/kvdb-rocksdb/src/lib.rs

```diff
@@ -425,7 +454,11 @@ impl Database {
 						}
 					}
 				}
-				db.write_opt(batch, &self.write_opts)?;
+
+				mark_corruption(
```

Comment:
I would rename `mark_corruption` into something like `check_for_corruption`, otherwise it looks like you're uncondtionnaly marking the db as corrupted here.

Reply:
Fixed 👍